CIP-0072/README.md

```diff
@@ -76,14 +72,14 @@ The on chain dApp registration certificate MUST follow canonical JSON and be ser
 ```
 
 ### Properties
-*`subject`*: Identifier of the claim subject (dApp). A UTF-8 encoded string. This uniqueness of this property cannot be guaranteed by the protocol and multiple claims for the same subject may exist, therefore it is required to exist some mechanism to assert trust in the *veracity* of this property.
+*`subject`*: Identifier of the claim subject (dApp). A UTF-8 encoded string, max 64 chars. The uniqueness of this property cannot be guaranteed by the protocol and multiple claims for the same subject may exist, therefore it is required to exist some mechanism to assert trust in the *veracity* of this property.
 
 *`type`*: The type of the claim. This is a JSON object that contains the following properties: 
 - *`action`*: The action that the certificate is asserting. It can take the following values: 
   - *`REGISTER`*: The certificate is asserting that the dApp is being registered for the first time. 
   - *`UPDATE`*: The certificate is asserting that the dApp is being updated.
   - *`DE_REGISTER`*: The certificate is asserting that the dApp version is being removed and it is requested that stores no longer show it.
-  - *`DE_REGISTER_ALL`*: The certificate is asserting that all dApp version are being removed and it is requested that stores no longer show it.
+  - *`DE_REGISTER_ALL`*: The certificate is asserting that the DApp's development is stopped, and it is depricated.So, no futher DApp's on-chain updats are expected.
```

Comment:
Do we want to change all "dApp" to "DApp" across document? @vhulchenko-iohk

Reply:
"The certificate is asserting that the dApp's development is stopped, and it is deprecated. So, no further dApp's on-chain update are expected."

Reply:
@matiwinnetou, yep, just inserted this exact line above.